doc/rst/language/spec/modules.rst

```diff
@@ -726,7 +677,7 @@ statements, or a combination of ``import`` and ``use`` statements, then the
 newly-visible names are inserted into a common enclosing scope.
 
 An error is signaled if multiple public module-level symbols would be inserted
-into this enclosing scope with the same name, and that name is referenced by
+into this enclosing scope with the same name, and that name is mentioned by
```

Comment:
The paragraphs present are simple enough that there won't be much saved by factoring them into a separate section, especially considering that use statements need to call out enums while import statements should not - we'll just end up putting more words in the shared section to indicate the different behavior.  This is the case even with an additional sentence about functions, which basically just links out to the Function Overloading section.

Reply:
> The paragraphs present are simple enough that there won't be much saved by factoring them into a separate section,

Not a big deal but my aim here is not to make the spec shorter but rather to make it more maintainable - and to make it easy to find and understand the differences and similarities between `import` and `use`. So it does not matter much to me whether there will be space savings.

Reply:
Sorry this is taking so long.  I'm trying to figure out where to put a section like that.  It seems like it relates to qualified naming, since that's one way to resolve conflicts, but that's not necessarily where you'd go looking for a section on it, you'd probably go looking for it either at the description of functions themselves (which I've got a link to in the commits that I haven't pushed yet) or at the description for things like variables.  But then the choice between the two is kinda arbitrary and we just moved all the use and import statement stuff out of the statements section and in short, I'm kinda stumped.  Do you have an idea of where you would like it?  And/or should I add it to the documentation issue and just push what I have for a final look?

Reply:
It makes sense to me to add it as a TODO in https://github.com/Cray/chapel-private/issues/1288